cb-sol-cli/cmd/rollup.js

```diff
@@ -0,0 +1,44 @@
+const {Command} = require('commander');
+const constants = require('../constants');
+const ethers = require('ethers');
+const {setupParentArgs, waitForTx, log, expandDecimals} = require("./utils")
+
+const transferCmd = new Command("transfer")
+    .description("execute transfer on rollupExample")
+    .option('--token <value>', 'TokenId to transfer', 10)
+    .option('--recipient <address>', 'Recipient', "")
+    .option('--rollupExample <address>', 'RollupExample contract address', "")
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+
+        const exampleInstance = new ethers.Contract(args.rollupExample, constants.ContractABIs.RollupExample.abi, args.wallet);
+        let recipient = args.recipient;
+        if (recipient == "") {
+            recipient = args.wallet.address;
+        }
+        log(args, `transfer tokens(${args.token}) to ${recipient} on contract ${args.rollupExample}`);
+        const tx = await exampleInstance.transfer(args.token, recipient);
+        await waitForTx(args.provider, tx.hash);
+    })
+
+const subRollupCmd = new Command("rollup")
+    .description("trigger rollup")
+    .option('--rollupExample <address>', 'RollupExample contract address', "")
+    .option('--batch <batch>', 'RollupExample contract address', 100)
+    .option('--destDomainId <domainId>', 'RollupExample contract address', 0)
+    .option('--resourceID <resourceID>', 'resourceID', "")
+    .action(async function (args) {
+        await setupParentArgs(args, args.parent.parent)
+
+        const exampleInstance = new ethers.Contract(args.rollupExample, constants.ContractABIs.RollupExample.abi, args.wallet);
+        const tx = await exampleInstance.rollupToOtherChain(args.destDomainId, args.resourceID, args.batch);
+        await waitForTx(args.provider, tx.hash);
+    })
+
+const rollupCmd = new Command("rollup")
+    .option('-d, decimals <number>', "The number of decimal places for the erc20 token", 18)
+    
+rollupCmd.addCommand(transferCmd)
+rollupCmd.addCommand(subRollupCmd)
+
+module.exports = rollupCmd
```

Comment:
Please add a new line at the end of file. This is a rule defined in the POSIX.

> Trailing newlines in non-empty files are a common UNIX idiom. Benefits of trailing newlines include the ability to concatenate or append to files as well as output files to the terminal without interfering with shell prompts.


References: 
- https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206
- https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_195
- https://eslint.org/docs/rules/eol-last